python/ql/src/Security/CWE-312/CleartextLogging.qhelp

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<include src="CleartextStorage.qhelp" /></qhelp>
```

Comment:
I can see that the JavaScript version of the query does this too, but it's rather weird to read about storage and cookies when the query is to do with logging. I don't suggest that you change this here, but think that it should be addressed in the future for both languages.

Reply:
I've raised a Jira issue in the QL project for this.